bin/mkdotenv

```diff
@@ -1,4 +1,4 @@
-#!/bin/sh
+# !/bin/sh
```

Comment:
We'll want to keep the `#` and `!` together on these scripts. They're not comments but [shebangs](https://en.wikipedia.org/wiki/Shebang_(Unix)).

Reply:
done